docs/pages/reference/config.mdx

```diff
@@ -716,6 +716,12 @@ proxy_service:
     - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
       cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
 
+    # Interval between attempts to reload the certificate key pairs.
+    # If one of the key pairs fails to load, then no key pair is reloaded.
+    # If set to 0, then periodic reloading is disabled.
+    # To use this feature correctly, certificate files should be updated atomically.
+    https_keypairs_reload_interval: 1h
```

Comment:
Should we include a comment on the fact the default for this is 0 (e.g disabled)

Reply:
Done in 95cb8dcee5ccc6087b4e61a7e0c22613394eb9af. Please let me know if it can be improved somehow.